src/actions/create.js

```diff
@@ -22,7 +22,7 @@ export let createEntryReducer = state => {
     contactPhoneInputValue: "",
     contactEmailInputValue: "",
     performanceInputValue: "Strong",
-    companyList: [newEntry, ...state.companyList.filter(company => company.name !== state.nameInputValue)]
+    companyList: [newEntry, ...state.companyList.filter(company => company.name !== state.companyEditSelect)]
```

Comment:
Changing name of a target created new entry due to nameInputValue no longer matching current entry. Now edits existing entry based on companyEditSelect.